lib/rubocop/formatter/disabled_config_formatter.rb

```diff
@@ -192,8 +192,8 @@ def filtered_config(cfg)
         # 'Enabled' option will be put into file only if exclude
         # limit is exceeded.
         rejected_keys = ['Enabled']
-        rejected_keys << 'EnforcedStyle' unless auto_gen_enforced_style?
-        cfg.reject { |key| rejected_keys.include?(key) }
+        rejected_keys << /^EnforcedStyle\w*/ unless auto_gen_enforced_style?
+        cfg.reject { |key| include_or_match?(rejected_keys, key) }
```

Comment:
Is there a logic by which the following would work, instead of creating the new `include_or_match` method?

```suggestion
        rejected_keys << 'EnforcedStyle' unless auto_gen_enforced_style?
        cfg.reject do |key|
          rejected_keys.any? { |rejected_key| key.start_with?(rejected_key) }
        end
```

Reply:
Yes, this would work too! However then it only works for prefixed strings of course. I tried to anticipate for possible use cases where a key would not necessarily just start with the rejected key.

Reply:
Yes, but any regexp could be used. :)